source/cl/examples/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ endif()
 if(${OCL_EXTENSION_cl_khr_command_buffer_mutable_dispatch})
   # Intentionally misnamed. Windows guesses that executables with "patch"
   # in the file name are installers and need admin privileges.
-  add_ca_cl_example(clMutableDispathcKHR
+  add_ca_cl_example(clMutableDispatchKHR
```

Comment:
This was not a typo, the comment explains why it had that name.

Reply:
Reverted this, thanks